`FutureWarning: Function stable_cumsum is deprecated`

Deprecation warning in CI. Aren't versions pinned? Will try rerunning.
- On #2581: https://github.com/opendp/opendp/actions/runs/20144560100/job/57822294389
```
FAILED test/extras/test_pca.py::test_pca_skl - opendp.mod.OpenDPException: 
  FFI("Continued stack trace from Exception in user-defined function:
Traceback (most recent call last):
  File "/home/runner/work/opendp/opendp/python/src/opendp/_convert.py", line 878, in wrapper_func
    py_out = func(py_arg)
  File "/home/runner/work/opendp/opendp/python/src/opendp/extras/sklearn/decomposition/__init__.py", line 353, in _postprocess
    ratio_cumsum = stable_cumsum(explained_variance_ratio_np)
  File "/opt/hostedtoolcache/Python/3.13.11/x64/lib/python3.13/site-packages/sklearn/utils/deprecation.py", line 95, in wrapped
    warnings.warn(msg, category=FutureWarning)
    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FutureWarning: Function stable_cumsum is deprecated; `sklearn.utils.extmath.stable_cumsum` is deprecated in version 1.8 and will be removed in 1.10. Use `np.cumulative_sum` with the desired dtype directly instead.
```

--- a/python/src/opendp/extras/sklearn/decomposition/__init__.py
+++ b/python/src/opendp/extras/sklearn/decomposition/__init__.py
@@ -320,7 +320,7 @@ if _decomposition is not None:
         def _postprocess(self, values):
             """A function that applies a release of the mean and eigendecomposition to self"""
             np = import_optional_dependency('numpy')
-            from sklearn.utils.extmath import stable_cumsum, svd_flip # type: ignore[import]
+            from sklearn.utils.extmath import svd_flip # type: ignore[import]
             from sklearn.decomposition._pca import _infer_dimension # type: ignore[import]
 
             self.mean_, S, Vt = values
@@ -350,7 +350,7 @@ if _decomposition is not None:
                 # their variance is always greater than n_components float
                 # passed. More discussion in issue: https://github.com/scikit-learn/scikit-learn/pull/15669
                 explained_variance_ratio_np = explained_variance_ratio_
-                ratio_cumsum = stable_cumsum(explained_variance_ratio_np)
+                ratio_cumsum = np.cumsum(explained_variance_ratio_np)
                 n_components = np.searchsorted(ratio_cumsum, n_components, side="right") + 1
 
             # Compute noise covariance using Probabilistic PCA model

